base/_functions.scss

```diff
@@ -12,3 +12,46 @@
 	}
 	@return $map;
 }
+
+
+// Bourbon changes (assumed) px values to EM or REM.
+// Here we change a unitless value to px
+// @jhogue
+@function px($value) {
+	// Make sure it is unitless
+	$value: strip-units($value);
+	@return ($value * 1px);
+}
+
+
+
+// Used for responsive typography and managing breakpoint values
+// @requires: SASS map of $breakpoints
+// @requires: SASS map of $type-sizes
+// @usage: For an EM value: bp(name)
+// @usage: For an PX value: bppx(name)
+// @usage: For a corresponding text size (from base/_textsize.scss): rwd-size(element,group)
+@function bp($size: 'base') {
+	@return em( map-get( $breakpoints, $size ) );
+}
+
+@function bppx($size: 'base') {
+	@return px( map-get( $breakpoints, $size ) );
+}
+
+@function rwd-size($elem, $group: 'base') {
```

Comment:
I think we can consolidate this now that we have the sass `deep-map-get()` helper, but that could come later.